Modules/ITS/src/ITSFhrTask.cxx

```diff
@@ -307,22 +318,10 @@ void ITSFhrTask::createOccupancyPlots() // create general plots like error, trig
       mStaveHitmap[istave] = new THnSparseI(Form("Occupancy/Layer%d/Stave%d/Layer%dStave%dHITMAP", mLayer, istave, mLayer, istave), Form("Hits on Layer %d, Stave %d", mLayer, istave), nDim, nBinstmp, Min, Maxtmp);
       getObjectsManager()->startPublishing(mStaveHitmap[istave]);
     }
-    if (mLayer == 3) {
-      mDeadChipPos = new TH2D(Form("Occupancy/Layer%d/Layer%dDeadChipPos", mLayer, mLayer), Form("DeadChipPos on Layer %d", mLayer), 28, etabinsOB3, 96, phibinsOB3);
-      mAliveChipPos = new TH2D(Form("Occupancy/Layer%d/Layer%dAliveChipPos", mLayer, mLayer), Form("AliveChipPos on Layer %d", mLayer), 28, etabinsOB3, 96, phibinsOB3);
-    } else if (mLayer == 4) {
-      mDeadChipPos = new TH2D(Form("Occupancy/Layer%d/Layer%dDeadChipPos", mLayer, mLayer), Form("DeadChipPos on Layer %d", mLayer), 28, etabinsOB4, 120, phibinsOB4);
-      mAliveChipPos = new TH2D(Form("Occupancy/Layer%d/Layer%dAliveChipPos", mLayer, mLayer), Form("AliveChipPos on Layer %d", mLayer), 28, etabinsOB4, 120, phibinsOB4);
-    } else if (mLayer == 5) {
-      mDeadChipPos = new TH2D(Form("Occupancy/Layer%d/Layer%dDeadChipPos", mLayer, mLayer), Form("DeadChipPos on Layer %d", mLayer), 49, etabinsOB5, 168, phibinsOB5);
-      mAliveChipPos = new TH2D(Form("Occupancy/Layer%d/Layer%dAliveChipPos", mLayer, mLayer), Form("AliveChipPos on Layer %d", mLayer), 49, etabinsOB5, 168, phibinsOB5);
-    } else {
-      mDeadChipPos = new TH2D(Form("Occupancy/Layer%d/Layer%dDeadChipPos", mLayer, mLayer), Form("DeadChipPos on Layer %d", mLayer), 49, etabinsOB6, 192, phibinsOB6);
-      mAliveChipPos = new TH2D(Form("Occupancy/Layer%d/Layer%dAliveChipPos", mLayer, mLayer), Form("AliveChipPos on Layer %d", mLayer), 49, etabinsOB6, 192, phibinsOB6);
-    }
+    mDeadChipPos = new TH2D(Form("Occupancy/Layer%d/Layer%dDeadChipPos", mLayer, mLayer), Form("DeadChipPos on Layer %d", mLayer), nHicPerStave[mLayer]*2, -0.5 * mLength[mLayer], 0.5 * mLength[mLayer], NStaves[mLayer]*7, -180, 180);
```

Comment:
I have fixed the binning for OB, it should be something like that (you can see new plots in your layout):

mDeadChipPos = new TH2D(Form("Occupancy/Layer%d/Layer%dDeadChipPos", mLayer, mLayer), Form("DeadChipPos on Layer %d", mLayer), nHicPerStave[mLayer]*7*0.5, -0.5 * mLength[mLayer], 0.5 * mLength[mLayer], NStaves[mLayer]*4, -180, 180)